bridge/src/main/java/io/stargate/bridge/service/ValuesHelper.java

```diff
@@ -166,14 +166,19 @@ public static ResultSet processResult(
       BiFunction<List<Column>, List<ByteBuffer>, io.stargate.db.datastore.Row> makeRow,
       RowDecorator rowDecorator)
       throws StatusException {
+
+    QueryOuterClass.ResumeMode resumeMode =
```

Comment:
Yea we have two options, adding a value `UNKNOWN` with the zero for example.. Or whatever.. 

Or wrap in the value wrapper as we have for example in the consistency and how it's done in this PR.. I am fine with both options..

@mpenick @olim7t any suggestions?

Reply:
Yes, [enums default to the zero value](https://developers.google.com/protocol-buffers/docs/proto3#default).

I think it would be cleaner to use a wrapper like we did for consistency.
I would even use null or an optional rather than `UNRECOGNIZED` to represent "unset" in the rest of the code. `UNRECOGNIZED` indicates a corrupt frame, so arguably we should error out rather than treating it as unset.

Reply:
Adapting to `null` as suggested by @olim7t.. I also feel `UNRECOGNIZED` is odd..